staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -184,8 +184,11 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.StorageConfig.HealthcheckTimeout, "etcd-healthcheck-timeout", s.StorageConfig.HealthcheckTimeout,
 		"The timeout to use when checking etcd health.")
 
-	fs.Int64Var(&s.StorageConfig.LeaseReuseDurationSeconds, "lease-reuse-duration-seconds", s.StorageConfig.LeaseReuseDurationSeconds,
+	fs.Int64Var(&s.StorageConfig.LeaseManagerConfig.ReuseDurationSeconds, "lease-reuse-duration-seconds", s.StorageConfig.LeaseManagerConfig.ReuseDurationSeconds,
 		"The time in seconds that each lease is reused. A lower value could avoid large number of objects reusing the same lease. Notice that a too small value may cause performance problems at storage layer.")
+
+	fs.Int64Var(&s.StorageConfig.LeaseManagerConfig.MaxObjectSize, "lease-max-object-size", s.StorageConfig.LeaseManagerConfig.MaxObjectSize,
```

Comment:
MaxObjectSize suggest that it's about size of the object.
Instead what we want is the number of object to be attached to a single lease.

So maybe MaxObjectCount ?